### ISSUE TITLE
Add GemCAD export for .gem files

### ISSUE THREAD

**Issue by geoffwatson** [mastercutter/hackagem#15]
It would be nice to be able to export designs for import into GemCAD

**Comment by mastercutter** [comment]
Good idea. Would exporting using the  .asc  file format work for you as well?   The .gem file format is not documented for public use as far as I know.

**Comment by kyletreece** [comment]
This feature would be really nice. .asc files would work great for me.

**Comment by mastercutter** [comment]
This is now work in progress. I will need some help to test it at some point.

**Comment by kyletreece** [comment]
Happy to test when ready, just let me know!
